### PR TITLE
Allow configuring the currency via env var

### PR DESCRIPTION
**Pull request by frm**
This requires us to set up `PROTOCOL_CURRENCY` on the secrets for the staging build on GitHub

### CODE REVIEW

File: .github/workflows/plugin_staging.yml [hunk 1]
```diff
@@ -56,6 +56,7 @@ jobs:
       REACT_APP_FRACTAL_WEBSITE_URL: ${{secrets.FRACTAL_WEBSITE_STAGING_URL}}
       REACT_APP_MAGURO_URL: ${{secrets.MAGURO_STAGING_URL}}
       REACT_APP_PROTOCOL_RPC_ENDPOINT: ${{secrets.PROTOCOL_RPC_STAGING_ENDPOINT}}
+      REACT_APP_PROTOCOL_CURRENCY: ${{secrets.PROTOCOL_CURRENCY}}
```
Comment (shelbyd) [inline .github/workflows/plugin_staging.yml:59]: Must we have a value set for this? What happens if we don't? I appreciate the ability to override, but would like to leave it as an override.

**Review by shelbyd — APPROVED** [review]

File: .github/workflows/plugin_staging.yml (hunk 1 above)
Comment (frm) [reply .github/workflows/plugin_staging.yml:59]: Defaults to FCL: https://github.com/trustfractal/fractal/pull/101/files#diff-d8cff6b1a75040dd7092aa1c3d03ddd6c36c9086c7f2ad8fd0b9f466f5d67db4R20

Comment (frm) [reply .github/workflows/plugin_staging.yml:59]: Should we change this to be "testFCL" or is it good as "FCL"?

Comment (shelbyd) [reply .github/workflows/plugin_staging.yml:59]: It's good as FCL.

**Pull request merged**